compute/client_library/requirements.txt

```
@@ -1,5 +1,5 @@
-isort==5.12.0; python_version > "3.7"
-isort==5.11.4; python_version <= "3.7"
+isort==5.13.2; python_version > "3.7"
+isort==5.13.2; python_version <= "3.7"
```

5.12 drops support for 3.7, so re-pin. https://github.com/pycqa/isort/blob/main/CHANGELOG.md#5120-january-28-2023
```suggestion
isort==5.11.5; python_version <= "3.7"
```